src/java_bytecode/java_bytecode_convert_method.h

```diff
@@ -6,6 +6,9 @@ Author: Daniel Kroening, kroening@kroening.com
 
 \*******************************************************************/
 
+/// \file
```

Comment:
Romain can confirm but this was replaced with the string_refinementt parameters AFAIK